packages/bygger-backend/src/services/formioService.ts

```diff
@@ -26,7 +28,43 @@ export class FormioService {
     return response.data as T;
   }
 
-  async getForm(formPath: string, type: FormType = 'form') {
+  async createNewForm(skjemanummer: string, userToken: string) {
+    const trimmedSkjemanummer = skjemanummer.trim();
+    const form: NavFormType = {
```

Comment:
Flytt til shared-domain

Reply:
Eventuelt se om man bør lage sak for å lage via backend

Reply:
flyttet til shared-domain